QUEST_LV_0100.tsv

```diff
@@ -693,9 +693,9 @@ QUEST_LV_0100_20150317_000692	$Guardian of the Kingdom's Secret Treasures
 QUEST_LV_0100_20150317_000693	You are qualified to receive the secret treasures.
 QUEST_LV_0100_20150317_000694	{memo X}
 QUEST_LV_0100_20150317_000695	$The magic suppressor of the tomb can be destroyed using the activating stones found inside the Guardians.{nl}The suppressed magic of the tomb will unleash and suppress the vicious energy.
-QUEST_LV_0100_20150317_000696	$The evil man left only demons' names to the Guardians.{nl}The Guardians will destroy the tomb without knowing that there is the Revelator.{nl}
-QUEST_LV_0100_20150317_000697	$Stop the final Guardian, Bearkaras, before he destroys the Royal Mausoleum.
-QUEST_LV_0100_20150317_000698	$Stop the final Guardian, Bearkaras, before he destroys the Royal Mausoleum.{nl}
+QUEST_LV_0100_20150317_000696	$This evil presence left only names of the demons to the Guardians.{nl}The Guardians will destroy the Royal Mausoleum without knowing that there is the Revelator.{nl}
```

Comment:
"without knowing that there is a Relevator" or "without knowing that the Relevator is here" ?